content/dart/concepts/queue/terms/contains/contains.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.contains()'
+Description: 'Checks if a specified element is present in the queue.'
+Subjects:
+ -'Code foundations'
+ -'Computer science'
+Tags:
+ -'Dart'
+ -'Concepts'
+ -'Queue'
+CatalogContent:
+ -'learn-dart'
+ -'paths/computer-science'
+---
+
+The Dart **.contains()** method in the 'Queue' class is used to check if a specified element is present in the queue. 
+
+## Sintax:
+
+The syntax for this method is as follows:
+
+```pseudo
+bool contains(Object element)
+```
+- `element`: The element is the object that needs to be checked in the queue.
+
+## Example-1:
+
+```dart
+var queue = Queue();
+queue.addAll([1, 2, 3, 4, 5]);
+print(queue.contains(3)); // Output will be true
```

Comment:
```suggestion
import 'dart:collection';

void main() {
  var queue = Queue();
  queue.addAll([1, 2, 3, 4, 5]);
  print(queue.contains(3)); 
}
```
You need to import the collection library and wrap the code inside the main function.